### ISSUE TITLE
configureScope setUser does not work in Electron main

### ISSUE THREAD

**Issue by xmedeko** [getsentry/sentry-electron#321]
## Important Details

How are you running Sentry?
Saas (sentry.io)

Sentry SDK: JS @sentry/electron@2.4.0

## Description

In Electron main process `Sentry.configureScope` with `scope.setUser` does not work when initiated by Electron IPC. When I check issues on https://sentry.io I see "Unknown User"

However, it works when `Sentry.configureScope` is called on the main process start.

## Steps to Reproduce

I do 
```js
    Sentry.configureScope(scope => {
        scope.setUser({
            id: "...",
            email: "..."
        });
        scope.setTag("company", "...");
    });
```

with some test data just after the `Sentry.init` in the main process, it works OK. 

However, the app get's the real user data in the BrowserWindow, then sends these data by Electron IPC to the main process and the main process (`ipcMain.on(...)`) calls `Sentry.configureScope`. Then, I see just "Unknown User" , but the tag "company" is set properly.

When I try debug that, it works OK. When Is use `Sentry.setUser(...)` (without `configureScope`) is works OK, too.

### What you expected to happen

I expect the user to be set by `Sentry.configureScope` always.

**Comment by timfish** [comment]
Hi @xmedeko. I can look into this but there are quite a few ways to configure and use Electron and I need a bit more context. The simplest way to do this is to supply a minimal example that reproduces the issue you are seeing.

**Comment by xmedeko** [comment]
I have Electron 12 with `contextIsolation:true, nodeIntegration:false` and I've hit the same bug as https://github.com/getsentry/sentry-electron/issues/316#issuecomment-827504173, that `Sentry.configureScope` does not send scope from browser to main. So, I've sent user info from browser by a custom `ipcRenderer.send` (exposed function in `preload.js`) and then `ipcMain.on` handler called `Sentry.configureScope(scope => { scope.setUser(...) })`.

**Comment by timfish** [comment]
There will shortly be a release of 2.4.1 that fixes #316

**Comment by xmedeko** [comment]
That's great! Anyway, it's strange this scenario fails.
